packages/lumx-react/src/components/button/IconButton.tsx

```diff
@@ -12,11 +12,16 @@ export interface IconButtonProps extends BaseButtonProps {
     icon: string;
     /**
      * Label text (required for a11y purpose).
-     * If you really don't want a tooltip and aria-label, you can give an empty label (this is not recommended).
+     * If you really an aria-label, you can set an empty label (this is not recommended).
      */
     label: string;
-    /** Props to pass to the tooltip (minus those already set by the IconButton props). */
-    tooltipProps?: Omit<TooltipProps, 'label'>;
+    /**
+     * Props to pass to the tooltip (minus those already set by the IconButton props).
+     * If undefined or if tooltipProps.label is undefined, the label prop will be used as tooltip label.
+     * */
+    tooltipProps?: TooltipProps;
+    /** Whether the tooltip should be displayed or not. */
+    showTooltip?: boolean;
```

Comment:
Wouldn't it be better to have a `hideTooltip` instead? What do you think @gcornut?